cli/internal/runsummary/format_json.go

```diff
@@ -29,11 +29,6 @@ func (rsm *Meta) FormatJSON() ([]byte, error) {
 }
 
 func (rsm *Meta) normalize() {
-	for _, t := range rsm.RunSummary.Tasks {
-		t.EnvVars.Global = rsm.RunSummary.GlobalHashSummary.envVars
-		t.EnvVars.GlobalPassthrough = rsm.RunSummary.GlobalHashSummary.passthroughEnvVars
-	}
-
```

Comment:
This was getting _super_ noisy; this gets output at the global level now.

Reply:
@mehulkar might have some thoughts here 😄

Reply:
my goal was to be able to compare task summaries without needing any extra logic to pull in other data. bit sad to see this get harder with this change.